### PR TITLE
chore: rename module name and upgrade ratify package to v1.2.1

### PR DESCRIPTION
**Pull request by junczhu**
- [X] Rename module name and upgrade ratify package to v1.2.1
- [X] Bump-up packages in go.mod
- [X] Test with Ratify CLI and K8S scenarios.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,7 +99,7 @@ Next, users will need to have the plugin within their Ratify pod in order to use
 One possible method to distribute plugins is by building a custom Ratify image
 
 ```Dockerfile
-FROM ghcr.io/deislabs/ratify:v1.0.0-beta.2 AS ratify
+FROM ghcr.io/ratify-project/ratify:v1.0.0-beta.2 AS ratify
```
Comment (susanshi) [inline README.md]: Update this to Ratify 1.3 once its available.

Comment (junczhu) [reply README.md]: Would do this in a separate PR

File: README.md [hunk 2]
```diff
@@ -126,7 +126,7 @@ image:
 Create a `Verifier` resource to register your custom plugin
 
 ```yaml
-apiVersion: config.ratify.deislabs.io/v1alpha1
+apiVersion: config.ratify.ratify-project.io/v1alpha1
```
Comment (binbin-li) [inline README.md]: CRD is not updated yet.

**Review by binbin-li — APPROVED** [review]

**Comment by susanshi** [comment]
This PR is pending validation results

**Comment by junczhu** [comment]
Standalone verification passed:
image

**Comment by susanshi** [comment]
Pending cli verification

**Comment by junczhu** [comment]
Pending local k8s test

**Comment by junczhu** [comment]
> Pending cli verification

DONE
image

**Comment by junczhu** [comment]
K8S verification DONE
image

**Review by binbin-li — APPROVED** [review]
lgtm, thanks for the validation!

**Review by susanshi — APPROVED** [review]
thanks for the validation results!

**Pull request merged**